src/makerspace/templates/makerspace/statistics.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'web/base.html' %}
+{% load static %}
+{% load i18n %}
+
+
+{% block title %}{% translate "Statistics" %}{% endblock title %}
+
+{% block extra_head %}
+    <script defer src="{% static 'makerspace/js/statistics_charts.js' %}"></script>
+{% endblock extra_head %}
+
+{% block body%}
+<div class="ui container">
+    <h1 class="ui text-centered make-col-blue header"> {% translate "Statistics" %}</h1>
+    <div class="ui column cards">
+    <div class="card " style="height: 50vh; width: 65vh">
+            <div class="content">
+            <div class="header">{% translate "How long each 3D-printer* has been reserved in total:" %} </div>
```

Comment:
I find "Totalt hvor lenge hver symaskin har blitt reservert" a bit complicated. I would suggest writing this in a simpler way, for example "Total reservasjonstid for 3D-printere" in Norwegian and "Total reservation time for 3D printers" in English. Same goes for sewing machines.